cupyx/fallback_mode/notification.py

```diff
@@ -0,0 +1,104 @@
+"""
+Methods related to notifications.
+"""
+
+import warnings
+import threading
+
+_thread_locals = threading.local()
+_thread_locals.dispatch_type = 'warn'
+_thread_locals.callback = None
+
+FallbackWarning = type('FallbackWarning', (Warning,), {})
+warnings.simplefilter(action='always', category=FallbackWarning)
```

Comment:
Don't initialize it in module-level code.
Please define the following function and call it at the end of this file.
```py
def _init_notification():
    FallbackWarning = type('FallbackWarning', (Warning,), {})
    warnings.simplefilter(action='always', category=FallbackWarning)
```

Reply:
Okay!

Reply:
`FallbackWarning` will be undefined here: https://github.com/cupy/cupy/pull/2279/files/8cd648b60966d25b0481e6e5f7403165db98be07#diff-7effd25fbbed4702a252a94a8a6e3cc5R58

Reply:
Done